space/authz/authz_test.go

```diff
@@ -195,6 +196,10 @@ func (r *resourceRepo) Delete(ctx context.Context, ID uuid.UUID) error {
 	return nil
 }
 
+func (r *resourceRepo) Exists(ctx context.Context, ID string) (bool, error) {
+	return false, nil
```

Comment:
Oh, never mind. The function looked exactly like the others. I didn't notice it was a test.